### ISSUE TITLE
Clarify that all contests are subject to audit

### ISSUE THREAD

**Issue by nealmcb** [FreeAndFair/ColoradoRLA#595]
ColoradoRLA lets the Audit Center put contests in two categories.

* It selects some contests for mandatory risk-limiting audit, and these "drive" the audit, determining how much sampling and interpretation is needed to establish that they have all met their risk limits, and thus when the audit is finished.

* The other contests are still audited to some degree, since the Audit Board enters vote interpretations for all contests on the sampled ballots, which can be analyzed to see whether risk limits have been attained even for those contests that have not been selected for mandatory risk limiting audit. When risk limits aren't met, risk levels could be measured.

Some of the text in the UI lists the second category of contests as "not selected for audit". It would seem more accurate to label them as "opportunistic audit", or with some other description that emphasizes that it is being audited, but just not _driving_ the audit.

**Comment by kiniry** [comment]
As I mentioned on Slack, CDOS asked us to remove mention of "opportunistic audit" from other UI elements.  We can certainly talk with them about this and if they want us to make a UI change during UI/UX polishing after EO training, we can do so.

**Comment by kiniry** [comment]
We'll get clarity on this from CDOS in September, so I've moved this issue to the next milestone.

**Comment by sfsinger19103** [comment]
Where exactly does this text appear? If it is only in the Contest Status table, then perhaps redesign of that table will make this question moot. See [#619](https://github.com/FreeAndFair/ColoradoRLA/issues/619)

**Comment by nealmcb** [comment]
It at least used to be in the the "Contest Updates" section of the SoS Home page: `client/src/component/sos/SoSHomePage.tsx:`
`        const status = sos.auditedContests[c.id] ? 'Under audit' : 'Not selected for audit';`

**Comment by sfsinger19103** [comment]
@nealmcb I agree with your point about language. I've raised the point with CDOS, thus far to no avail.
